### PR TITLE
Add Julia taiwan into communities list

### PR DESCRIPTION
**Pull request by yuehhua**
我們是一個新興的程式語言社群 Julia Taiwain，這邊是我們的社群資訊，有些 slide 我可能後續補完。

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
    - [GCPUG.TW](#gcpug.tw)
    - [Golang Taiwan](#golang-taiwan)
    - [Scala Taiwan](#scala-taiwan)
+   - [Julia Taiwan](#julia-taiwan)
```
Comment (sufuf3) [inline README.md]: Would you please move this line between `[Golang Taiwan](#golang-taiwan)` & `[Scala Taiwan](#scala-taiwan)`.
Let's let TOC in alphabetical order.

File: README.md [hunk 2]
```diff
@@ -49,6 +50,13 @@
 - Location: 台灣(Taiwan)
 - Keywords: `Scala`
 
+### Julia Taiwan
```
Comment (sufuf3) [inline README.md]: Would you please also follow the alphabetical order?
Many thanks.

**Comment by yuehhua** [comment]
@sufuf3 Here it is!

File: communities/JuliaTaiwan.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+###### `julialang`, `HPC`, `Scientific computing`, `Numerical computing`
+
+## Table of Contents
+- [介紹](#介紹)
+- [活動投影片](#活動投影片)
+
+
+## 介紹
+Julia是一個新興的高階、高效動態語言，有逼近 C 的效能，兼有 Python 的優雅語法。以高效科學運算為原點，發展成一種通用語言 (general-purpose language)，得力於LLVM，加上語法本身優雅而精巧的設計，促成了高效的特性。Julia支援分散式運算 (distributed computing)、平行運算 (parallelism)、共時 (concurrency)等特性，讓使用者更易於擴充 (scale up)。Julia支援命令式、物件導向、函數式、泛型等等程式設計典範，讓這個語言更加豐富。
```
Comment (iGene) [inline communities/JuliaTaiwan.md]: Suggest to follow this guideline:
https://github.com/sparanoid/chinese-copywriting-guidelines
To add a whitespace character between Chinese and English characters

**Comment by iGene** [comment]
LGTM

**Pull request merged**